bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
yeah, seems like it to me too, but I thought I might be missing something. unless @pmclanahan knows something I don't, the 'product' key doesn't seem to be used at all here?